src/main/java/mate/academy/springboot/swagger/util/StringParser.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class StringParser {
```

Comment:
```suggestion
@Component
public class SortUtil {
```
add this as field in controller